### ISSUE TITLE
[MaterialDialog] Padding

### ISSUE THREAD

**Issue by RishitKedia** [hectahertz/react-native-material-dialog#24]
Hey! 👋 

This is great! Thank you for working on this. 🙏 

I have a use-case where I'd like to control the padding of `MaterialDialog` and set it to 0.

Negative margins work on iOS, but on Android, I'm not able to get past `paddingTop` set here: 😭 
[https://github.com/hectahertz/react-native-material-dialog/blob/9bd81fce3535c189b354f234a6b966c056f1c0ab/src/MaterialDialog.js#L107](https://github.com/hectahertz/react-native-material-dialog/blob/9bd81fce3535c189b354f234a6b966c056f1c0ab/src/MaterialDialog.js#L107)

## Possible Solutions

1. Adding a `modalContainerStyle` prop, and merging it with the `styles.modalContainer` View.

2. Instead of setting `paddingTop` in `styles.modalContainer`, setting it in `styles.contentContainer`, but then we'll have to set it in all the `styles.containers` individually.

Any thoughts on this? Thanks for your time! 😃

**Comment by hectahertz** [comment]
Hi @RishitKedia,

Thanks for your clear explanation! 😄

I think your use case can be quite common, so maybe we could just add a prop to remove all paddings and that would be an even cleaner solution, what do you think? 😀 

Thanks for using the library!

**Comment by RishitKedia** [comment]
Hey, @hectahertz! 👋

That would be awesome! Didn't think of that. 😛 Sounds great to me as a user of the library. 💯

Thanks for considering my request! 🙏

**Comment by hectahertz** [comment]
Awesome! 🦄

If you wanna prepare a pull request I'll review it and release it asap, otherwise I'll try to make some time work on it in the near future 👨‍💻

Thanks! 🙏

**Comment by RishitKedia** [comment]
Sure, I'd be more than happy to prepare a PR for this, but I looked into the code, and I'm not entirely sure as to how you'd want this to be done. 😰 

It would be really nice if you could find some time to work on this. I'd be super grateful. 😃

**Comment by hectahertz** [comment]
Hi @RishitKedia 

I started working on this and I can't really visualize the design where you need to set that padding to 0.

Do you think you could maybe provide some snippet or screenshot of the design so I can come up with a clean way to solve it?

Thanks! 🤗

**Comment by RishitKedia** [comment]
Hey, @hectahertz! 👋

Sure, here are two screenshots: 

## With Padding

<img width="375" alt="with-padding" src="https://user-images.githubusercontent.com/7033218/33515842-a83c5876-d781-11e7-8afb-c5507f4ccbdb.png">

## Without Padding

<img width="375" alt="without-padding" src="https://user-images.githubusercontent.com/7033218/33515851-de40713c-d781-11e7-8cc1-cf3647a839c2.png">

Achieved this using: 
```javascript
style={{
  marginLeft: -24,
  marginRight: -24,
  marginTop: -24,
  marginBottom: -24
}}
```

As stated in the OP, it works on iOS, but not fully on Android (because of `paddingTop`). 

---

Thanks for taking out the time to work on this! 🙏

**Comment by hectahertz** [comment]
Just released 0.6.4, you can set `addPadding={false}` now and it should be good 😄 

Let me know how it goes!

**Comment by RishitKedia** [comment]
Just tested `v0.6.4`, and it works! Woot! 🤘 

Thanks, Hector! 😃
